src/Microsoft.ML.FastTree/Representation/TreeRegressor.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System;
+using System.Linq;
+using Microsoft.ML.Trainers.FastTree.Internal;
+
+namespace Microsoft.ML.FastTree.Representation
+{
+    /// <summary>
+    /// A container class for exposing <see cref="RegressionTree"/>'s attributes to users.
+    /// This class should not be mutable, so it contains a lot of read-only members.
+    /// </summary>
+    public class TreeRegressor
```

Comment:
>TreeRegressor [](start = 17, length = 13)

Did anyone yet ask you about sample file in [https://github.com/dotnet/machinelearning/tree/master/docs/samples/Microsoft.ML.Samples/Dynamic](https://github.com/dotnet/machinelearning/tree/master/docs/samples/Microsoft.ML.Samples/Dynamic) ?
Would be nice to put  
```
/// <example>
        /// <format type="text/markdown">
        /// <![CDATA[
        ///  [!code-csharp[FastTree](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/TreeRegressor.cs)]
        /// ]]></format>
```
here so user can see how to use this class in documentation.

Reply:
Not yet until we have address [issue 2319](https://github.com/dotnet/machinelearning/issues/2319).

---
In reply to: [252087326](https://github.com/dotnet/machinelearning/pull/2243#discussion_r252087326) [](ancestors = 252087326)